src/actions/omnisharp-auto-complete-actions.el

```diff
@@ -276,10 +339,15 @@ triggers a completion immediately"
     (ignore-case omnisharp-company-ignore-case)
 
     (sorted omnisharp-company-sort-results)
+    ;; (sorted (if (eq omnisharp-company-match-type 'company-match-simple)
```

Comment:
Ah, good old pull requests and the absence of a version cutoff.

The flex matching code is a draft implementation and needs some tidying up. :)



On 22 February 2015 at 12:10:42, Mika Vilpas (notifications@github.com) wrote:

In src/actions/omnisharp-auto-complete-actions.el:

> @@ -276,10 +339,15 @@ triggers a completion immediately"
>      (ignore-case omnisharp-company-ignore-case)
>   
>      (sorted omnisharp-company-sort-results)
> +    ;; (sorted (if (eq omnisharp-company-match-type 'company-match-simple)
Do we need these?

—
Reply to this email directly or view it on GitHub.